src/Symfony/Component/Intl/Resources/emoji/build.php

```diff
@@ -143,20 +143,29 @@ public static function buildSlackRules(array $emojisCodePoints): iterable
         foreach ($emojis as $data) {
             $emojiCodePoints = str_replace('-', ' ', strtolower($data['unified']));
             $shortCode = $data['short_name'];
+            $shortCodes = $data['short_names'];
+            array_walk($shortCodes, function (&$value, $key) {
+                $value = ":$value:";
+            });
+
             if (!array_key_exists($emojiCodePoints, $emojisCodePoints)) {
                 $ignored[] = [
                     'emojiCodePoints' => $emojiCodePoints,
                     'shortCode' => $shortCode,
+                    'shortCodes' => $shortCodes,
                 ];
                 continue;
             }
             $emoji = $emojisCodePoints[$emojiCodePoints];
             self::testEmoji($emoji, 'slack');
             $codePointsCount = mb_strlen($emoji);
-            $maps[$codePointsCount][$emoji] = ":$shortCode:";
+            $maps['emoji-slack'][$codePointsCount][$emoji] = ":$shortCode:";
+            foreach ($shortCodes as $short_name) {
+                $maps['slack-emoji'][$codePointsCount][$short_name] = $emoji;
```

Comment:
I'd rather create 2 variables than having `$maps['emoji-slack']` and `$maps['slack-emoji']` as the actual maps